eclair-core/src/main/resources/reference.conf

```diff
@@ -121,6 +121,8 @@ eclair {
     sync {
       request-node-announcements = true // if true we will ask for node announcements when we receive channel ids that we don't know
       encoding-type = zlib // encoding for short_channel_ids and timestamps in query channel sync messages; other possible value is "uncompressed"
+      channel-range-chunk-size = 3500 // how many short channel ids we are going to answer at once; we can theoretically fit 4091 uncompressed channel ids in a single lightning message (max size 65 Kb)
```

Comment:
I'm not satisfied with the naming of both parameters. If anyone has a better idea...